src/froala-editor.js

```diff
@@ -4,7 +4,7 @@ import {ObserverLocator} from "aurelia-binding";
 import {Config} from './froala-editor-config';
 
 // Import Froala Editor
-import FroalaEditor from 'froala-editor/js/froala_editor.pkgd.min.js'
+import FroalaEditor from 'froala-editor/js/froala_editor.min.js'
```

Comment:
@Geet06  import FroalaEditor from 'froala-editor'; we should use this instead of js file.